docker-compose.yml

```diff
@@ -84,66 +84,6 @@ services:
       interval: 30s
       timeout: 20s
       retries: 3
-  elasticsearch:
```

Comment:
Ignore changes in this file. I need to revert this before merging.

Reply:
FYI This review won't let me merge, I just wanted to block myself from accidentally merging before I fixed this because I know I'm going to forget.